### ISSUE TITLE
cpplint not run on save

### ISSUE THREAD

**Issue by hsmyy** [wangzw/CppStyle#4]
Hi, I met a problem about running cpplint. When I save the file, the console outputs like this:
```
Cannot find clang-format configure file under any level parent directories of path (cpp file omitted).
Run clang-format with Google style by default.
Run clang-format command: /usr/local/Cellar/clang-format/2015-04-02/bin/clang-format -assume-filename=(cpp file omitted) -style=Google
```
And clang-format runs ok, but cpplint has no response, how can I find the problem?
Thank you very much.

**Comment by wangzw** [comment]
Hi,

It seems that you enable running both cpplint and clang-format on save. Since eclipse's console output is asynchronous, sometimes clang-format and cpplint will output to console at the same time and sometimes one will clear another.

So it's hard to say cpplint not work according to your console output. To test it, disable running clang-format on save and save your source file again after modification. You will see cpplint's output.


Another way to test cpplint after enabling both cpplint and clang-format on save. Remove the "copyright" on the top of your source code and save, if the cpplint report this issue correctly, it should work fine.

**Comment by hsmyy** [comment]
Thank you very much for your fast help.
Sorry after I tried as you said(turn off clang-format), there is still no response. no output on the console.
I am using Luna 4.4.2, is this version ok? I saw you tested on 4.4.1 which is ok.

p.s. clang-format works, your work is efficient and easy of use!

**Comment by wangzw** [comment]
Version: Luna SR1 (4.4.1)
Build id: M20140925-0400

I'm using this version and it works well. Which OS are you using? 

It supposes to work well on Luna 4.4.2 and Linux/Windows and MacOS. But I only test it on my mac.

According to your description, it should be a bug, I will take a look this issue later.

**Comment by hsmyy** [comment]
```
My OS is Mac 10.10.1
eclipse.buildId=4.4.2.M20150204-1700
```

**Comment by wangzw** [comment]
Hey, I reproduce the bug.

You try to enable running cpplint on save for all C/C++ projects in Eclipse preference page, right?

The good news is, you can enable cpplint for a C/C++ projects by configure the project's property page.

Right click the project in project explorer and click "Properties" and you will see.


![2015-05-07 9 21 42](https://cloud.githubusercontent.com/assets/1254826/7516076/2469feb6-f4ff-11e4-97a0-0e5b3083913e.png)

Hope this work around could work for you. I will fix the bug at this weekend.

**Comment by wangzw** [comment]
fixed in 1.2.2.0
